2018-08-07 15:00:47: 您的主机中的软件中止了一个已建立的连接。   at System.Net.Sockets.Socket.BeginReceive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags, AsyncCallback callback, Object state)
   at Yinrg.Net.Sockets.SocketClient.mSIdN55ykX2Ndb5boCu(Object , Object , Int32 , Int32 , SocketFlags , Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 15:00:47: 您的主机中的软件中止了一个已建立的连接。   at System.Net.Sockets.Socket.BeginReceive(Byte[] buffer, Int32 offset, Int32 size, SocketFlags socketFlags, AsyncCallback callback, Object state)
   at Yinrg.Net.Sockets.SocketClient.mSIdN55ykX2Ndb5boCu(Object , Object , Int32 , Int32 , SocketFlags , Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:05:18: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:05:18: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:05:18: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:05:18: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:05:18: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:05:18: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:05:18: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:05:18: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:05:18: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:07:52: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:20:39: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:37: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2018-08-07 16:23:38: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndSend(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.wQh1k8mXtn4yGgQ8LDh(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.SendByteCallBack(IAsyncResult ar)
2018-08-07 16:24:06: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 16:35:18: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:01:03: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:01:03: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:01:03: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:01:03: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:01:03: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:01:03: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:01:03: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:01:03: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:01:03: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:01:03: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:01:03: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:01:03: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:01:03: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:01:03: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:01:04: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:01:18: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:33:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:33:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:33:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:33:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:33:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:33:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:33:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:33:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:33:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:33:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:33:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:33:59: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:34:14: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:34:20: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:34:20: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:34:20: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:34:20: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:34:20: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:34:20: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:34:20: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:34:20: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:34:25: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2018-08-07 17:34:26: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
